### PR TITLE
Dynamic config fixes

### PR DESCRIPTION
**Pull request by regisb**
This PR includes various fixes related to dynamic configuration:

- Fix Tutor filter deprecation warning
- Make sure MFE image is pushed to docker.io in CI
- Update k8s docs
- Make sure lms container is started before the mfe containers
- Fix profile MFE url
- Disable learner records
- Lower MFE config API cache timeout

Each change is indicated in a separate commit.

This PR should be merged in the olive branch as well.

### CODE REVIEW

**Comment by regisb** [comment]
FYI @arbrandes @ghassanmas @kdmccormick. If you test the MFE dynamic configuration you'll almost certainly need those changes.

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 variables:
     TUTOR_PLUGIN: mfe
-    TUTOR_BUILD_IMAGES: mfe
+    TUTOR_IMAGES: mfe
```
Comment (arbrandes) [inline .gitlab-ci.yml:3]: I meant to ask how the images were going to be built.  +1

File: tutormfe/patches/local-docker-compose-prod-services [hunk 2]
```diff
@@ -4,3 +4,5 @@ mfe:
     build:
         context: ../plugins/mfe/build/mfe/
     restart: unless-stopped
+    depends_on:
+        - lms
```
Comment (arbrandes) [inline tutormfe/patches/local-docker-compose-prod-services:8]: The change makes total sense, but I'm curious: did you actually run into a failure without it?  It seemed to worked on my local tests, but maybe I wasn't fast enough. :)

File: tutormfe/patches/openedx-lms-development-settings [hunk 3]
```diff
@@ -19,12 +19,13 @@ CSRF_TRUSTED_ORIGINS.append("{{ MFE_HOST }}:{{ app["port"] }}")
...
-ENABLE_MFE_CONFIG_API = True
-
+# https://openedx.github.io/frontend-platform/module-Config.html
 MFE_CONFIG = {
     "BASE_URL": "{{ MFE_HOST }}",
-    "CREDENTIALS_BASE_URL": "/csrf/api/v1/token",
+    {%- if MFE_PROFILE_MFE_APP %}
+    "ACCOUNT_PROFILE_URL": "http://{{ MFE_HOST }}:{{ MFE_PROFILE_MFE_APP["port"] }}",
+    {%- endif %}
+    "CREDENTIALS_BASE_URL": "",
```
Comment (arbrandes) [inline tutormfe/patches/openedx-lms-development-settings:28]: Nice catch!

**Review by arbrandes — APPROVED** [review]
Thanks @regisb!

File: tutormfe/patches/local-docker-compose-prod-services (hunk 2 above)
Comment (regisb) [reply tutormfe/patches/local-docker-compose-prod-services:8]: No I didn't, but I would have if I had run `tutor dev start account`... Which I probably already did one day. MFEs depended on the LMS even before dynamic config, so this is not a new failure case.

**Pull request merged**